src/payload.cc

```diff
@@ -25,6 +25,7 @@
 // OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 #include "payload.h"
+#include "triton/common/logging.h"
```

Comment:
Is this import necessary? It doesn't look you added any logging calls.

Reply:
This is from debugging, will remove